src/containers/SubjectViewer.jsx

```diff
@@ -457,14 +457,18 @@ class SubjectViewer extends React.Component {
 SubjectViewer.propTypes = {
   dispatch: PropTypes.func,
   //--------
+  splits: PropTypes.object,
+  user: PropTypes.shape({
+    id: PropTypes.string
+  }),
+  //--------
   currentSubject: PropTypes.shape({
     src: PropTypes.string,
   }),
+  subjectStatus: PropTypes.string,
```

Comment:
Is this prop being used by the SubjectViewer?

Reply:
Wait, derp, I moved that logic to fetchSubject() already.